src/goto-instrument/cover_filter.cpp

```diff
@@ -43,6 +43,34 @@ bool internal_functions_filtert::operator()(
   return true;
 }
 
+/// Filter out all functions except those defined in the file that is given
+/// in the constructor.
+/// \param function: the function under consideration
+/// \param goto_function: a goto function
+/// \return returns true if `function` is defined in the file
+/// given in the constructor
+bool file_filtert::operator()(
+  const symbolt &function,
+  const goto_functionst::goto_functiont &goto_function) const
+{
+  (void)goto_function; // unused parameter
```

Comment:
Put the documentation to the declaration, then you don't need this at the implementation because you can leave the parameter name off

Reply:
I'd prefer to keep this as it it, since I was just re-using the style the file had before my changes.